tests/test_nestedclasses.py

```
@@ -310,36 +304,37 @@ def remove(self, item: Item, amount: int = 1):
                     self.items.remove(stack)
             if amount == 0:
                 break
-        if amount :
+        if amount:
             raise Exception("Not enough items in inventory")
         else:
             return
-        
+
     def __str__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)